src/Tools/Source/RunTests/ProcessTestExecutor.cs

```diff
@@ -81,7 +82,16 @@ private string GetResultsFilePath(AssemblyInfo assemblyInfo)
                 // an empty log just in case, so our runner will still fail.
                 File.Create(resultsFilePath).Close();
 
-                var dumpOutputFilePath = Path.Combine(resultsDir, $"{assemblyInfo.DisplayName}.dmp");
+                // Define environment variables for processes started via ProcessRunner.
+                var environmentVariables = new Dictionary<string, string>();
+                environmentVariables.Add(ProcDumpRunner.ProcDumpPathEnvironmentVariableKey, _options.ProcDumpPath);
```

Comment:
Oh, I see what you mean: I read this out of context where this is originally setting it for child processes but we're also mangling ourself too...yeah, that seems less than great.